src/Common/src/System/Net/Logging/EmailEventSource.cs

```diff
@@ -10,18 +10,18 @@
 
 namespace System.Net
 {
-    [EventSource(Name = "Microsoft-System-Net-Mail", LocalizationResources = "FxResources.System.Net.Mail.SR")]
-    internal sealed class WebEventSource : EventSource
+    [EventSource(Name = "Microsoft-System-Net-Mail", Guid = "32663b9f-4d8d-44f8-9f29-f4efc12e6879", LocalizationResources = "FxResources.System.Net.Mail.SR")]
```

Comment:
@Priya91 @vancem 

It's fine to remove the GUID but add it within both the source code as well as https://github.com/dotnet/corefx/blob/master/Documentation/debugging/windows-instructions.md#traces

> However you can also use *Microsoft-System-Net-Mail in most tools instead (which is MUCH nicer).

I found this doesn't work `logman`. 
@vancem please review https://github.com/dotnet/corefx/blob/master/Documentation/debugging/windows-instructions.md#traces and let me know if there is a way to use the `*<name>` format using existing Windows tools.